doc/releases/quincy.rst

```diff
@@ -5,6 +5,539 @@ Quincy
+* pybind/mgr: object_format.py decorator updates & docs (`pr#47979 <https://github.com/ceph/ceph/pull/47979>`_, John Mulligan)
+* pybind/mgr: tox and test fixes (`pr#49508 <https://github.com/ceph/ceph/pull/49508>`_, Kefu Chai)
+* pybind/mgr: use memory temp_store for sqlite3 db (`pr#50286 <https://github.com/ceph/ceph/pull/50286>`_, Patrick Donnelly)
+* pybind/rados: notify callback reconnect (`pr#48113 <https://github.com/ceph/ceph/pull/48113>`_, Nitzan Mordechai)
+* python-common: Add 'KB' to supported suffixes in SizeMatcher (`pr#48242 <https://github.com/ceph/ceph/pull/48242>`_, Tim Serong)
+* qa/cephadm: remove fsid dir before bootstrap in test_cephadm.sh (`pr#47949 <https://github.com/ceph/ceph/pull/47949>`_, Adam King)
+* qa/fs/mixed-clients: specify distros for tests (`pr#49957 <https://github.com/ceph/ceph/pull/49957>`_, Dhairya Parmar)
+* qa/suites/rbd: fix sporadic "rx-only direction" test failures (`pr#50113 <https://github.com/ceph/ceph/pull/50113>`_, Ilya Dryomov)
+* qa/suites/rgw: fix and update tempest and barbican tests (`pr#50002 <https://github.com/ceph/ceph/pull/50002>`_, Tobias Urdin)
+* qa/tasks/cephadm.py: fix pulling cephadm from git.ceph.com (`pr#49858 <https://github.com/ceph/ceph/pull/49858>`_, Adam King)
+* qa/tasks/kubeadm: set up tigera resources via kubectl create (`pr#48080 <https://github.com/ceph/ceph/pull/48080>`_, John Mulligan)
+* qa/tasks/rbd_fio: bump default to fio 3.32 (`pr#48386 <https://github.com/ceph/ceph/pull/48386>`_, Ilya Dryomov)
+* qa/tests: added quincy client upgrade => reef (`pr#50353 <https://github.com/ceph/ceph/pull/50353>`_, Yuri Weinstein)
+* qa/tests: initial draft for quincy p2p tests (`pr#46896 <https://github.com/ceph/ceph/pull/46896>`_, Yuri Weinstein, Laura Flores)
+* qa/workunits/rados: specify redirect in curl command (`pr#49140 <https://github.com/ceph/ceph/pull/49140>`_, Laura Flores)
+* qa/workunits/windows: backport rbd-wnbd tests (`pr#49883 <https://github.com/ceph/ceph/pull/49883>`_, Lucian Petrut)
+* qa: Fix test_subvolume_group_ls_filter_internal_directories (`pr#48327 <https://github.com/ceph/ceph/pull/48327>`_, Kotresh HR)
+* qa: Fix test_subvolume_snapshot_info_if_orphan_clone (`pr#48325 <https://github.com/ceph/ceph/pull/48325>`_, Kotresh HR)
+* qa: ignore disk quota exceeded failure in test (`pr#48164 <https://github.com/ceph/ceph/pull/48164>`_, Nikhilkumar Shelke)
+* qa: switch back to git protocol for qemu-xfstests (`pr#49544 <https://github.com/ceph/ceph/pull/49544>`_, Ilya Dryomov)
+* qa: switch to https protocol for repos' server (`pr#49471 <https://github.com/ceph/ceph/pull/49471>`_, Xiubo Li)
+* qa: wait for scrub to finish (`pr#49459 <https://github.com/ceph/ceph/pull/49459>`_, Milind Changire)
+* Quincy: osd/scrub: use the actual active set when requesting replicas… (`pr#48543 <https://github.com/ceph/ceph/pull/48543>`_, Ronen Friedman)
```

Comment:
This commit seems to be an odd one out. Not sure why it isn't with other osd commits.

Reply:
I fixed this.